packages/parser/lib/css_lexer.re

```diff
@@ -552,13 +552,13 @@ let rec get_next_token = buf => {
   | combinator => COMBINATOR(latin1(buf))
   | string => STRING(latin1(~skip=1, ~drop=1, buf))
   | important => IMPORTANT
-  | not_operator => NOT_OPERATOR(latin1(buf))
-  | (or_operator, whitespace) => OR_OPERATOR(latin1(buf))
-  | and_operator => AND_OPERATOR(latin1(buf))
-  | only_operator => ONLY_OPERATOR(latin1(buf))
   | equal_sign => EQUAL_SIGN(latin1(buf))
-  | greater_than => GREATER_THAN(latin1(buf))
-  | less_than => LESS_THAN(latin1(buf))
+  | not_operator => NOT_OPERATOR(latin1(buf))
```

Comment:
Would it make more sense to have a not_operator always come with a WS after?

This way we can remove the func: from the parser.

Reply:
`func` rule is no more